app/scripts/lib/channels/mixins/postmessage_receiver.js

```diff
@@ -1,12 +1,12 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-'use strict';
 
 /**
  * A channel mixin that expects responses to sent messages. Responses
  * are expected to be received via a postMessage to the window.
  */
+ 'use strict';
```

Comment:
Why this change?
And does moving the `'use strict'` beneath the comment break any JSDoc-type-functionality (if we're even using JSDoc for this repo)? /cc @vladikoff